src/Component/Menu/index.js

```diff
@@ -0,0 +1,25 @@
+
+import { NavLink, Outlet } from "react-router-dom";
+import './Menu.css';
+
+function Menu() {
+
+  const linkClass = (isActive) => {
+    console.log(isActive);
```

Comment:
Should remove this `console.log`; readability.

Reply:
Fixed this thing ... no problem batman-duck!